docs/src/main/asciidoc/spanner-spring-data-r2dbc.adoc

```diff
@@ -0,0 +1,50 @@
+:spring-data-commons-ref: https://docs.spring.io/spring-data/data-commons/docs/current/reference/html
+
+[#spring-data-cloud-spanner-r2dbc]
+== Cloud Spanner Spring Data R2DBC
+
+The Spring Data R2DBC Dialect for Cloud Spanner enables the usage of https://github.com/spring-projects/spring-data-r2dbc[Spring Data R2DBC] with Cloud Spanner.
+
+The goal of the Spring Data project is to create easy and consistent ways of using data access technologies from Spring Framework applications.
+
+=== Setup
+
+Maven coordinates, using link:getting-started.xml#bill-of-materials[Spring Framework on Google Cloud BOM]:
+
+[source,xml]
+----
+<dependencies>
+    <!-- The starter dependency for the Spring Data R2DBC framework. -->
+    <dependency>
+        <groupId>org.springframework.boot.experimental</groupId>
```

Comment:
This is also outdated, not experimental anymore.
```suggestion
        <groupId>org.springframework.boot</groupId>
```
In fact, it's part of `spring-boot-dependencies` [here](https://github.com/spring-projects/spring-boot/blob/v2.7.13/spring-boot-project/spring-boot-dependencies/build.gradle#L1571). So the starter version corresponds to Spring Boot version.

Reply:
sure, updated it